src/client/pages/Debugger/ActualDebugger.tsx

```diff
@@ -27,7 +30,9 @@ export const ActualDebugger: React.FC = () => {
           {(storage) => (
             <>
               <Button
+                disabled={isCleaningQuoteState}
                 onClick={() => {
+                  setIsClearningQuoteState(true)
```

Comment:
Is anything resetting this to false? 🤔

Reply:
🤦🏻‍♂️ Good catch. I mean, it's working because at the end of this function we force a reload. But definitely I should have reset it to `false`

Reply:
Ahh right, understand. But just because it's confusing when reading the code it might be worth implementing it hehe